src/App.css

```diff
@@ -1,38 +1,3 @@
-.App {
+.app {
   text-align: center;
```

Comment:
- Kindly, change the layout of your project to have two columns and add a detail page as the previous code reviewer [suggested](https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/pull/19#discussion_r1247464918) to stick the given design and have better UI 👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/52010789/2080b630-6a0e-4edc-a9de-e3cfd76c4eb0" width="200">|<img src="https://github.com/DilsherB/covid-data-capstone-project-with-react-redux/assets/52010789/67edfe3c-d0da-4a92-bb80-74e2a44f5756" width="200">|


😎️

🤝️

Reply:
done